examples/vumps/vumps_ising.jl

```diff
@@ -102,3 +102,72 @@ Sz2_infinite = expect(ψ.AL[2] * ψ.C[2], "Sz")
 
 @show Sz1_finite, Sz2_finite
 @show Sz1_infinite, Sz2_infinite
+
+##############################################################################
+#Extract a finite MPS from an InfiniteCanonicalMPS by slicing
+
+function finite_mps(ψ::InfiniteCanonicalMPS, range::AbstractRange)
+  @assert isone(step(range))
+  ψ_finite = ψ.AL[range]
+  ψ_finite[last(range)] *= ψ.C[last(range)]
+  l0 = linkind(ψ.AL, first(range) - 1 => first(range))
+  lN = linkind(ψ.AR, last(range) => last(range) + 1)
+  ψ_finite = MPS([δ(l0, dag(l0)'); [ψ_finiteᵢ for ψ_finiteᵢ in ψ_finite]; δ(dag(lN), lN')])
+  set_ortho_lims!(ψ_finite, (last(range) + 1):(last(range) + 1))
+  return ψ_finite
+end
```

Comment:
Let's move this into the package (say to `src/infinitecanonicalmps.jl`) and export it (add it to the export list in `src/ITensorInfiniteMPS.jl`).